src/igvfd/audit/analysis_set.py

```diff
@@ -165,6 +165,8 @@ def audit_analysis_set_files_missing_analysis_step_version(value, system):
     files_with_missing_asv = []
     for file in files:
         file_object = system.get('request').embed(file + '@@object?skip_calculated=true')
+        if file_object.get('derived_manually') is True:
```

Comment:
Since `file_object.get('derived_manually')` is a boolean you can omit the `is True` part at the end.